lib/purchases_flutter.dart

```diff
@@ -159,8 +163,15 @@ class Purchases {
   /// [customerInfoUpdateListener] CustomerInfo update listener.
   static void addCustomerInfoUpdateListener(
     CustomerInfoUpdateListener customerInfoUpdateListener,
-  ) =>
-      _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+  ) {
+    _customerInfoUpdateListeners.add(customerInfoUpdateListener);
+    if (_customerInfoUpdateListeners.length == 1) {
```

Comment:
Why do this only if there's one listener?

Reply:
I believe the preferred approach was to only send the customer info on the first added listener. But if we think we should send it to all listeners as soon as we add them, I can change it easily.

Reply:
Note that this will also send it when we remove all listeners and add one again.

Reply:
> But if we think we should send it to all listeners as soon as we add them, I can change it easily.

I think so. That's how it works in iOS with the delegate or with the `CustomerInfo` `AsyncStream`.

Reply:
Well, in iOS we can only have one delegate. It would send the last customer info if they changed it, but we can't have multiple.

That said, I'm not sure of the use cases for people to add multiple listeners, so I'm not sure what the desired behavior would be... But well, it might make sense for consistency to send the customer info for all listeners as soon as they are added, so I will change it. CC @aboedo @vegaro